doc/docs/api/examples.md

```diff
@@ -1,21 +1,46 @@
 <!-- markdownlint-disable-file MD013 MD024 -->
 # Examples using the public D-Bus API
 
-## GoLang
+These steps were based on CentOS Stream 9
 
-First, make sure the system has golang installed and download the dbus library.
+## Ruby
 
-These steps were based on CentOS Stream 9:
+### Installing ruby and dbus module
 
-### Installing golang
+``` bash
+dnf install ruby -y
+gem install ruby-dbus
+```
+
+### Build the source code
 
 ``` bash
-# dnf install go-toolset
+# ruby list_nodes.rb
+BlueChi nodes:
+================
+Name: control
+Path: online
+Status: /org/eclipse/bluechi/node/control
+
+Name: node1
+Path: offline
+Status: /org/eclipse/bluechi/node/node1:x
 ```
 
-### Adding dbus library
+### List all nodes
+
+``` ruby
+--8<-- "api-examples/ruby/list_nodes.rb"
+```
+
+---
+
+## GoLang
+
+### Installing golang and dbus module
 
 ``` bash
+# dnf install go-toolset
```

Comment:
What exactly is this package needed for?

Reply:
looks like as I added some lines above golang session it changed in the markdown?